### ISSUE TITLE
Penetration 2d output

### ISSUE THREAD

**Issue by kashkoulimohammad** [MTgeophysics/mtpy#128]
Dear all,

I tried to get a suitable output from plot_penetration_depth2d and I changed some parameters in the penetration.py and penetration_depth2d.py but none of the parameters changed the output.
How can I control output?
![MT_Penetration_Depth_Profile_by_det](https://user-images.githubusercontent.com/11463583/98440738-ba5fd200-210f-11eb-8009-c07d834f2089.png)

Regards,
Mohammad

**Comment by alkirkby** [comment]
I am assuming you would like to select fewer periods to plot, is that right? The example script plot_penetration_depth2d.py in examples/scripts shows how to do this. Could you please paste your script here so we can see what parameters you are using?

**Comment by kashkoulimohammad** [comment]
No, I want to plot all of the periods and control the legend but I could not. I used the script you mentioned.
